custom_components/ipx800v4/binary_sensor.py

```diff
@@ -36,26 +40,50 @@ async def async_setup_entry(
 
     for device in devices:
         if device.get(CONF_TYPE) == TYPE_VIRTUALOUT:
-            entities.append(VirtualOutBinarySensor(device, controller, coordinator))
+            on_value = 0 if device.get(CONF_INVERT) else 1
+            entities.append(VirtualOutBinarySensor(device, controller, coordinator, on_value))
         elif device.get(CONF_TYPE) == TYPE_DIGITALIN:
-            entities.append(DigitalInBinarySensor(device, controller, coordinator))
+            on_value = 0 if device.get(CONF_INVERT) else 1
+            entities.append(DigitalInBinarySensor(device, controller, coordinator, on_value))
 
     async_add_entities(entities, True)
 
 
 class VirtualOutBinarySensor(IpxEntity, BinarySensorEntity):
     """Representation of a IPX Virtual Out."""
 
+    def __init__(
+        self,
+        device_config: dict,
+        ipx: IPX800,
+        coordinator: DataUpdateCoordinator,
+        on_value: int
+    ) -> None:
+        """Initialize the VirtualOutBinarySensor."""
+        super().__init__(device_config, ipx, coordinator)
+        self.on_value = on_value
+
     @property
     def is_on(self) -> bool:
         """Return the state."""
-        return self.coordinator.data[f"VO{self._id}"] == 1
+        return self.coordinator.data[f"VO{self._id}"] == self.on_value
```

Comment:
and after that:
```suggestion
        return self.coordinator.data[f"VO{self._id}"] == 1 if not self._inverted_value else 0
```

Reply:
this introduced a bug that I fixed in https://github.com/Aohzan/ipx800/pull/48/commits/89fd13fa5e1b5e4fb63b81cdc4aded4dda0837e3